### PR TITLE
Adding user-agent info

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

This PR adds information about how users should now use the user-agent header instead of the browser variable.

### Preview

https://docs-staging.datadoghq.com/margot.lepizzera/syn-user-agent/synthetics/guide/identify_synthetics_bots/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 1]
```diff
@@ -70,6 +81,10 @@ You can also leverage your API and browser test configuration **Advanced options
 
 ## Browser variable
 
+<div class="alert alert-warning">
+In some cases, the browser variable cannot be applied. This method is deprecated and it is now recommended to leverage the `user-agent` header instead.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:85]:
```suggestion
In some cases, the browser variable cannot be applied. The browser variable method is consequently deprecated and it is now recommended to leverage the `user-agent` header instead.
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 2]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
 
-The following header is attached to all single and multistep API tests:
+For single and multistep API tests, the `user-agent` header is `Datadog/Synthetics`.
 
-`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{% /tab %}}
+{{% tab "Browser tests" %}}
 
-A `user-agent: Datadog/Synthetics` is also added.
+For browser tests, the value of the `user-agent` header varies depending on the browser and device executing the test. The `user-agent` header value however always ends with `Datadog/Synthetics` to allow you to easily identify our tests.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:54]:
```suggestion
For browser tests, the value of the `user-agent` header varies depending on the browser and device executing the test. The `user-agent` header value however always ends with `Datadog/Synthetics` to allow you to easily identify Synthetic tests.
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 3]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
 
 You can also identify Datadog robots by using some of the **default headers** attached to the requests generated by Synthetic tests:
 
+### user-agent
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:42]:
```suggestion
### `user-agent`
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 4]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
 
-The following header is attached to all single and multistep API tests:
+For single and multistep API tests, the `user-agent` header is `Datadog/Synthetics`.
 
-`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{% /tab %}}
+{{% tab "Browser tests" %}}
 
-A `user-agent: Datadog/Synthetics` is also added.
+For browser tests, the value of the `user-agent` header varies depending on the browser and device executing the test. The `user-agent` header value however always ends with `Datadog/Synthetics` to allow you to easily identify Synthetic tests.
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:54]:
```suggestion
For browser tests, the value of the `user-agent` header varies depending on the browser and device executing the test. The `user-agent` value always ends with `Datadog/Synthetics` to allow you to identify Synthetic tests.
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 5]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
 
 {{% /tab %}}
-{{% tab "Browser tests" %}}
+{{< /tabs >}}
 
-The following header is attached to all browser tests:
+### sec-datadog
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:59]:
```suggestion
### `sec-datadog`
```

Comment (ruthnaebeck) [reply content/en/synthetics/guide/identify_synthetics_bots.md:59]: Is there an explanation of what this is?

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 6]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
 
-`Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{< tabs >}}
+{{% tab "Single and multistep API tests" %}}
+
+`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:64]: This is taking up multiple lines on the docs page. Use 3 ticks for a code block.

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 7]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
+
+{{% /tab %}}
+{{% tab "Browser tests" %}}
 
-A `user-agent` header with a value that varies depending on the type of browser test run (browser, device) is also added.
+`Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:69]: This is taking up multiple lines on the docs page. Use 3 ticks for a code block.

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 8]
```diff
@@ -70,6 +81,10 @@ You can also leverage your API and browser test configuration **Advanced options
 
 ## Browser variable
 
+<div class="alert alert-warning">
+In some cases, the browser variable cannot be applied. The browser variable method is consequently deprecated and it is now recommended to leverage the `user-agent` header instead.
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/identify_synthetics_bots.md:85]:
```suggestion
The browser variable is deprecated. Datadog recommends using the `user-agent` header instead.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 9]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
+{{< tabs >}}
+{{% tab "Single and multistep API tests" %}}
+
+`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+
+{{% /tab %}}
+{{% tab "Browser tests" %}}
 
-A `user-agent` header with a value that varies depending on the type of browser test run (browser, device) is also added.
+`Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:69]:
```suggestion
```
Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>
```
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 10]
```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
...
+{{< /tabs >}}
 
-The following header is attached to all browser tests:
+### sec-datadog
 
-`Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{< tabs >}}
+{{% tab "Single and multistep API tests" %}}
+
+`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:64]:
```suggestion
```
sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>
```
```

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 6 above)
Comment (MargotLepizzera) [reply content/en/synthetics/guide/identify_synthetics_bots.md]: Fixed - resolving comment.

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 7 above)
Comment (MargotLepizzera) [reply content/en/synthetics/guide/identify_synthetics_bots.md]: Fixed - resolving comment.

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 5 above)
Comment (MargotLepizzera) [reply content/en/synthetics/guide/identify_synthetics_bots.md]: Not really. I still went ahead and added a small sentence to introduce it though.

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 11]
```diff
@@ -70,6 +87,10 @@ You can also leverage your API and browser test configuration **Advanced options
 
 ## Browser variable
 
+<div class="alert alert-warning">
+The browser variable is deprecated. Datadog recommends using the `user-agent` header instead.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:91]:
```suggestion
The browser variable is deprecated. Datadog recommends using the user-agent header instead.
```

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**